spec/services/contacts/filter_service_spec.rb

```diff
@@ -40,22 +43,37 @@
       it 'filter contacts by additional_attributes' do
         params[:payload] = payload
         result = filter_service.new(params, user_1).perform
-        expect(result.length).to be 2
+        expect(result[:contacts].length).to be 1
```

Comment:
Can we add a bunch of contacts with a mix of attributes to ensure that we don't break these filters in future? 
- for example, having a contact here with the same name, but without the browser language.

Reply:
also let's validate the objects in the results[:contacts] array and ensure the id matches our intended contact.